components/script/dom/webidls/HTMLInputElement.webidl

```diff
@@ -41,7 +41,8 @@ interface HTMLInputElement : HTMLElement {
   //         attribute DOMString step;
            attribute DOMString type;
            attribute DOMString defaultValue;
-  [TreatNullAs=EmptyString] attribute DOMString value;
+           [TreatNullAs=EmptyString, SetterThrows]
```

Comment:
It doesn't should it be indented to a different level?

Here's what it looks like in `whitespace-mode`:

<img width="415" alt="screen shot 2016-02-19 at 6 59 13 pm" src="https://cloud.githubusercontent.com/assets/226605/13192514/eb412a6e-d73a-11e5-94d8-f2e618a46247.png">